cli-toolchain/quickstart.md

```diff
@@ -272,30 +272,20 @@ Behind the scenes, the CLI automates populating of the resource identifiers for
 You can use a Lambda function to call your GraphQL API, however at this time you must modify your AppSync schema after deploying the API. For example, deploy a simple `Todo` model with the following schema in the `amplify add api` flow:
 
 ```
-type Todo @model {
+type Todo @model @auth (
+    rules: [
+        { allow: private, provider: iam, operations: [create] }
+    ]
+) {
   id: ID!
   name: String
   description: String
 }
 ```
 
-Next, run `amplify console` and select the GraphQL API. On the **Settings** page add an **Additional authorization provider** and select **IAM**. Next, on the **Schema** page add in the `@aws_iam` directive where appropriate per the [AppSync documentation](https://docs.aws.amazon.com/appsync/latest/devguide/security.html#using-additional-authorization-modes). For example if you just want your Lambda function to have access to run a single mutation, add the directive onto that mutation (`createTodo` below) as well as the return type:
+In the above example we want your Lambda function to have access to run a single mutation (`createTodo`) and hence we explicitly mention `create` in the `operations` list.
```

Comment:
Can you add to the end of this something like "To grant access for application users to perform other actions, such as read only you would add another auth rule. See XXX link for more examples"

Reply:
Added.